src/test/java/meditracker/medication/MedicationManagerTest.java

```diff
@@ -3,13 +3,24 @@
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 
+/**
+ * This test file is to perform tests on the MedicationManager
+ */
+
 public class MedicationManagerTest {
 
+    /**
+     * Resets the MedicationManager
+     *
+     * @throws InvocationTargetException when it fails to call the reset method
+     * @throws IllegalAccessException when the method can not access the specified class
+     * @throws NoSuchMethodException
+     */
+
```

Comment:
Shift this javadoc to after the start of the `//@@author` tag, and also remove the blank line

Reply:
Got it. Resolved in [9a106bf](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/240/commits/9a106bf99990df8b1eb65f1b78a133c32ce486b8).